internal/commands/document.go

```diff
@@ -204,10 +126,18 @@ func getDocumentation(path string, severity string, outputDirectory string) ([]D
 
 		regoWithoutComments := getRegoWithoutComments(policy.Contents)
 		document := Document{
-			Header:   header,
-			Severity: trimContent(severity),
-			URL:      trimContent(url),
-			Rego:     trimContent(regoWithoutComments),
+			Header:     header,
+			Severities: strings.Join(policy.RuleNames, " "),
```

Comment:
Will there be a case in which there is more than a single severity? I think it'd be technically possible by having a single header in a `src.rego` that contains both a `warn` and a `violation`-- but I'm not sure it would make a whole lot of sense to do so.

Reply:
I would not combine different severities into one policy, but it is possible. Given this tool is both targeted at GK/Kubernetes and is opinionated, perhaps we should throw an error if a given policy has more than one rule type rather than trying to handle all of these cases? I think that makes the most sense.

Reply:
I'm torn. Warn won't really do anything in GK except just bloat the template. But having warn in the same file could be what some people want to do. I tend to prefer not putting barriers up unless we really need to or it's something we can't recover from.

What I'm trying to rationalize is in what case(s) would it make sense from a documentation point of view. A policy doc that is both a violation and a warning seems confusing. They'd have to write out the warning case in the comment header